shared/Microsoft.AspNetCore.RangeHelper.Sources/RangeHelper.cs

```diff
@@ -30,7 +32,8 @@ internal static class RangeHelper
         public static (bool, RangeItemHeaderValue) ParseRange(
```

Comment:
Super unrelated to your PR, but could we give these tuple elements names - `(bool isRangeRequest, RangeItemHeaderValue range)`? It's not really clear what these values are meant to be.